docs/develop/chain/fullnode.md

```diff
@@ -14,8 +14,43 @@ They act as a backend for Websites, verify new blocks and validate extrinsics (e
 
 There are currently two different runtimes (i.e., two different parachain environments) that a KILT full node can be part of:
 
-- Peregrine: the public test network whose runtime is as close to the official chain as possible. It can be used to try stuff out before executing them on the live chain, which involves spending tokens that have real monetary value.
-- Spiritnet: the official public network, which contains only stable features.
+- **Spiritnet**: the official public network, which contains only stable and thoroughly-tested features.
+- **Peregrine**: the public test network whose runtime is as close to that of Spiritnet as possible. It can be used to try stuff out before executing them on the production Spiritnet chain, which involves spending tokens that have real monetary value.
```

Comment:
```suggestion
- **Peregrine**: the public test network whose runtime is as close to that of Spiritnet as possible.
It can be used to try features out before executing them on the production Spiritnet chain, which involves spending tokens that have a real monetary value.
```